packages/ui/Button/Button.js

```diff
@@ -51,6 +51,14 @@ const ButtonBase = kind({
 		 */
 		css: PropTypes.object,
 
+		/**
+		 * Additional DOM nodes which may be necessary for decorating the Button.
+		 *
+		 * @type {Node}
+		 * @private
```

Comment:
I would say "not at this time". I marked it private to sort of validate whether it was an acceptable approach. We can discuss this prop and its name at some point.

Reply:
Since we reference it in the `TooltipDecorator` docs, does it have to be public? If we leave it private, how will authors know to use it when attaching a Tooltip?

Reply:
No. It doesn't have to be public at all. It's marked private here. It's not really intended to be externally used at this time. If we discover a need for it in the future, I'm happy to open it up as public, as we've then established a broach and general need for it.

Reply:
I don't agree for the reasons above but I'm content to move ahead with it.